src/views/tickets/tpp.php

```diff
@@ -39,7 +39,7 @@ class="tribe-tickets-tpp cart <?php echo esc_attr( $messages_class ); ?>"
 	<input type="hidden" name="provider" value="Tribe__Tickets__Commerce__PayPal__Main">
```

Comment:
This view needs a `@since TBD __what changed__` and `@version TBD`

Reply:
done. good catch.